kythe/java/com/google/devtools/kythe/analyzers/base/KytheEntrySets.java

```diff
@@ -320,7 +320,10 @@ public void emitOrdinalEdges(
   /** Returns (and emits) a new abstract node over child. */
   public EntrySet newAbstractAndEmit(
       VName child, List<VName> params, @Nullable MarkedSource markedSource) {
-    NodeBuilder absBuilder = newNode(NodeKind.ABS).addSignatureSalt(child);
+    NodeBuilder absBuilder =
+        newNode(NodeKind.ABS)
+            .addSignatureSalt(child)
+            .setCorpusPath(new CorpusPath(child.getCorpus(), "", ""));
```

Comment:
Should this share the child's path/root as well?

Reply:
SGTM